### PR TITLE
rename timestamp related variables

### PR DESCRIPTION
**Pull request by netcriptus**
Implements https://github.com/raiden-network/raiden-contracts/issues/1580

### CODE REVIEW

File: raiden_contracts/data/source/services/UserDeposit.sol [hunk 1]
```diff
@@ -161,7 +161,7 @@ contract UserDeposit is Utils {
 
         withdraw_plans[msg.sender] = WithdrawPlan({
             amount: amount,
-            withdraw_timestamp: block.timestamp + withdraw_delay
+            withdrawable_after: block.timestamp + withdraw_delay
```
Comment (andrevmatos) [inline raiden_contracts/data/source/services/UserDeposit.sol:164]: Per the description of the issue, shouldn't we also rename `withdraw_delay` to `withdraw_timeout`? Does this variable match the `_timeout` usecase, or is it something different?

**Review by andrevmatos — CHANGES_REQUESTED** [review]
The changes here seem correct, but there're still variables which need renaming in the contracts; e.g. `expiration_time` in `TokenNetwork.setTotalWithdraw`; could you please give another pass and rename any variable or parameter name in any contract which should mach the new naming schema?

File: raiden_contracts/data/source/services/UserDeposit.sol (hunk 1 above)
Comment (netcriptus) [reply raiden_contracts/data/source/services/UserDeposit.sol]: Good call ✅

File: raiden_contracts/data/source/raiden/TokenNetwork.sol [hunk 2]
```diff
@@ -1596,7 +1596,7 @@ contract TokenNetwork is Utils, Controllable {
         uint256 channel_identifier,
         address participant,
         uint256 total_withdraw,
-        uint256 expiration_timestamp,
+        uint256 recoverable_until,
```
Comment (andrevmatos) [inline raiden_contracts/data/source/raiden/TokenNetwork.sol:1599]: This function is called directly with parameters from `WithdrawInput`, therefore I think the name here should be the same from the struct:
```suggestion
        uint256 withdrawable_until,
```

**Review by andrevmatos — COMMENTED** [review]
Just one change now, thanks!

File: raiden_contracts/data/source/raiden/TokenNetwork.sol (hunk 2 above)
Comment (netcriptus) [reply raiden_contracts/data/source/raiden/TokenNetwork.sol]: Nice catch ✅

**Review by andrevmatos — APPROVED** [review]
LGTM, nice refactoring, thank you

**Pull request merged**